TicketSwap/solar
# docs(guidelines): adding general guidelines

# Description

Added guidelines for installation and use of brand and colors

This is my reference: [BrandBook 2020](https://drive.google.com/file/d/1-7afSQaC5gIHUYx7jZvPa5n_A7WKS-ph/view)

## How to test

- Checkout this branch
- `$ yarn storybook`
- Have a look at the guidelines and confirm those `.mdx` files are working

## Screenshots

![image](https://user-images.githubusercontent.com/12168698/139438708-2d2cb60e-97ef-441f-bf0a-c5cb7edb9f6a.png)

![image](https://user-images.githubusercontent.com/12168698/139438863-7686341e-7c14-4d97-b9fb-8200449472a9.png)
